modules/virt-creating-single-pvc-template-storage-pool.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-single-pvc-template-storage-pool_{context}"]
+= Creating a storage pool using a `pvcTemplate` in a host path provisioner (HPP) custom resource.
```

Comment:
Remove backticks from `pvcTemplate` in title here, per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc:

> Do not use backticks or other markup in assembly or module headings. You can use backticks or other markup in the title for a block, such as a code block .Example or a table .Description title.

Reply:
Good to know. Done.